kotlin/codegen/pom.xml

```diff
@@ -56,32 +56,31 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.truth</groupId>
+      <artifactId>truth</artifactId>
+      <version>1.0</version>
+      <scope>test</scope>
+    </dependency>
 
     <!--
-      The Kotlin compiler must be near the end of the list because its .jar file includes an
+      The Kotlin compiler usage must be near the end of the list because its .jar file includes an
       obsolete version of Guava!
     -->
     <dependency>
-      <groupId>org.jetbrains.kotlin</groupId>
-      <artifactId>kotlin-compiler-embeddable</artifactId>
-      <scope>test</scope>
+      <groupId>me.eugeniomarletti.kotlin.metadata</groupId>
+      <artifactId>kotlin-metadata</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.jetbrains.kotlin</groupId>
-      <artifactId>kotlin-annotation-processing-embeddable</artifactId>
+      <groupId>com.github.tschuchortdev</groupId>
+      <artifactId>kotlin-compile-testing</artifactId>
+      <version>1.2.2</version>
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>me.eugeniomarletti.kotlin.metadata</groupId>
-      <artifactId>kotlin-metadata</artifactId>
-    </dependency>
-    <!--
-      Though we don't use compile-testing, including it is a convenient way to get tools.jar on the
-      classpath. This dependency is required by kapt3.
-    -->
-    <dependency>
-      <groupId>com.google.testing.compile</groupId>
-      <artifactId>compile-testing</artifactId>
+      <groupId>com.squareup.okio</groupId>
+      <artifactId>okio</artifactId>
+      <version>2.1.0</version>
```

Comment:
versions should go in `<dependencyManagement>` section of root pom.

Reply:
Part of a4c55f7